packages/ui/src/theme/css/component/inAppMessaging/banner.scss

```diff
@@ -0,0 +1,49 @@
+.amplify-in-app-messaging-banner__button {
+  background-color: var(--amplify-components-bannermessage-button-background-color);
+  border-style: none;
+  width: var(--amplify-components-bannermessage-button-width);
+}
+.amplify-in-app-messaging-banner__buttons-container {
+  margin-top: var(--amplify-components-bannermessage-buttons-container-margin-top);
+}
+.amplify-in-app-messaging-banner__container {
+  background-color: var(--amplify-components-bannermessage-banner-container-background-color);
+  box-shadow: var(--amplify-components-bannermessage-banner-container-box-shadow);
+  flex-direction: column;
+  margin: var(--amplify-components-bannermessage-banner-container-margin);
+  min-width: 400px;
+  padding: var(--amplify-components-bannermessage-banner-container-padding);
+  position: absolute;
+  right: 0;
+  &.xl, &.large, &.medium {
+      max-width: 50%;
+  }
+  &.base, &.small {
+      width: calc(100% - 32px);
```

Comment:
Any specific reason to use `32px` over `2rem`?

Reply:
Nope!